plugins/mysql/mysqlembed.cpp

```diff
@@ -675,11 +675,16 @@ class MySQLPreparedStatement : public CInterface
     {
         if (stmt && *stmt)
         {
+            // NOTE - we ignore all but the first result from stored procedures
```

Comment:
If the results need to be freed even if not requested then this code should be after the call.  Good enough for a point fix though - that can be separately investigated.